WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -292,14 +292,14 @@ class PageListViewModel @Inject constructor(
                                 ScheduledPage(
```

Comment:
wdyt about adding named parameters here for the first 3 params? I know it's not necessary but given the subtitle optional param has been placed right below title, it now looks weird to have most of the parameters in named fashion with only the first few with a different notation. Also, comparing these two it looks a lot clearer to have names for everything given it's quite a list of parameters to look into :)

Reply:
good idea 👍 , I'll change it in all the places